snowflake/identity_store.go

```diff
@@ -109,12 +109,18 @@ func (s *IdentityStoreSyncer) SyncIdentityStore(ctx context.Context, identityHan
 			displayName = userRow.Name
 		}
 
+		isMachine := false
+		if strings.EqualFold(userRow.Type, "SERVICE") || strings.EqualFold(userRow.Type, "LEGACY_SERVICE") {
+			isMachine = true
+		}
```

Comment:
Maybe we can just do this in one line?
```isMachine := strings.EqualFold(userRow.Type, "SERVICE") || strings.EqualFold(userRow.Type, "LEGACY_SERVICE")```

Reply:
Lol. Painful that I didn't do that :) Pushed it now.